oauth2/migrations/sql/tests/cockroach/1_test.sql

```diff
@@ -0,0 +1,53 @@
+-- +migrate Up
```

Comment:
Same here :)

Reply:
Same, it has the contents of `9_test.sql` replacing `9-` by `1-` as it's the first migration for cockroach.